src/components/Modal/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Modal } from './Modal';
+export type { ModalProps } from './Modal';
```

Comment:
We have 2 exports `ModalProps ` and `Modal`, but the `ModalProps` is not used in components expose. And there are only 2 exports in `./Modal`. So we can use it `export * from './Modal';`

Reply:
I agree with @joonchoi0918

Reply:
I am personally not a huge fan of `export * from './Modal'` - I like to keep exports very explicit. Although, `export *` may be fine since someone could just do `import * from 'zui/Modal/Modal` and skip the index anyway 🤷‍♂️ Will see what @jtulloch says.

Reply:
On second thought, yeah I think `export *` is ok here. My initial hesitation with them is we would have to be careful with what we export from `Modal`, but with either method we have to be careful with one or the other so it's arbitrary.

Reply:
agreed. i think it's fine here.